packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -61,11 +61,13 @@ integration with social identity providers such as Facebook, Google, Amazon, Mic
 SAML.
 
 Using the CDK, a new user pool can be created as part of the stack using the construct's constructor. You may specify
-the `userPoolName` to give your own identifier to the user pool. If not, CloudFormation will generate a name.
+the `userPoolName` to give your own identifier to the user pool. If not, CloudFormation will generate a name. As shown
+below, setting usernames to be case insensitive is preferred in most situations.
```

Comment:
This line doesn't have any relationship to the preceding paragraph, so seems out of place here. I think the whole "case sensitive" story deserves its own paragraph, no? Especially with the comment line below and its "most situations" qualifier, please explain the consequences. Also explain that the setting cannot be changed after creation (without replacing the pool which deletes the user database).

Reply:
The preceding paragraph described the userPoolName parameter in the example, so I think it's pretty reasonable to also describe the signInCaseSensitive parameter I addd to the example. No? But I can add some explanation in a new paragraph - I'm thinking after the example.